### PR TITLE
Remove deprecated methods in vNext

### LINKED ISSUE #2238: next: Remove deprecated methods
Here is a list of remaining deprecated methods I could find in the `next` branch
- pasteUssFile: `packages/zowe-explorer/src/uss/actions.ts`
- copyDataSet: `packages/zowe-explorer/src/dataset/actions.ts`
- copyUssFile: `packages/zowe-explorer-api/src/tree/IZoweTreeNode.ts`

### PR DESCRIPTION
**Pull request by t1m0thyj**
## Proposed changes

Resolves #2238 by removing methods that were deprecated in v2.

Also consolidates VS Code commands to remove duplication.

## Release Notes

Milestone: v3

Changelog: Removed deprecated methods and consolidated VS Code commands (see file for more details)

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

File: packages/zowe-explorer-api/__mocks__/vscode.ts
Comment (traeok) [inline packages/zowe-explorer-api/__mocks__/vscode.ts:933]: Was this removed from VS Code types? I'm curious to know why this was deleted from mocks 😋

File: packages/zowe-explorer/src/configuration/Constants.ts
Comment (traeok) [inline packages/zowe-explorer/src/configuration/Constants.ts:25]: Thanks for your efforts on greatly reducing the number of commands we have 😁

**Review by traeok — APPROVED** [review]
LGTM, thanks @t1m0thyj!

**Review by zFernand0 — APPROVED** [review]
I love how much some tests were cleaned after the consolidation in this PR 🥳 
LGTM! 😋

**Review by zFernand0 — DISMISSED** [review]
Given that we are in a Porting-PR phase (#2936), I believe it would be good to try and merge this PR quickly before other port PRs are opened. The other alternative is to place this PR on hold until all other port PRs are merged.

In either scenario, this PR LGTM! 😋

File: packages/zowe-explorer-api/__mocks__/vscode.ts
Comment (traeok) [reply packages/zowe-explorer-api/__mocks__/vscode.ts:933]: Good catch, thanks for updating that!

**Review by traeok — APPROVED** [review]

**Review by zFernand0 — DISMISSED** [review]
LGTM! 😋 

---

Audit can be safely ignored for now.
- See: https://github.com/zowe/zowe-explorer-vscode/pull/2950#pullrequestreview-2111142797

File: packages/zowe-explorer-api/CHANGELOG.md
Comment (JillieBeanSim) [inline packages/zowe-explorer-api/CHANGELOG.md:18]: Could we add this Breaking changes list to the branch's ./docs/early-access/v3/extenders.md file also please?

File: packages/zowe-explorer/CHANGELOG.md
Comment (JillieBeanSim) [inline packages/zowe-explorer/CHANGELOG.md:34]: same as other CHANGELOG comment, please add to docs file in branch for repo.

**Review by JillieBeanSim — CHANGES_REQUESTED** [review]
Thanks @t1m0thyj for working on this! I do see 2 files being edited when running command `pnpm run --if-present --parallel vscode:prepublish`.  Also left comment on documenting breaking changes.

File: packages/zowe-explorer-api/CHANGELOG.md
Comment (t1m0thyj) [reply packages/zowe-explorer-api/CHANGELOG.md:18]: I think going forward we want to document changes in the Box document and publish them on Zowe.org, but since that's quite new and users may be expecting to find the list in Extenders.md, I've updated it too for now 👍

**Review by traeok — APPROVED** [review]

**Review by JillieBeanSim — DISMISSED** [review]
Thanks @t1m0thyj this looks great!

**Review by zFernand0 — DISMISSED** [review]
LGTM! 😋 
Seems like there are a few merge conflicts 😢

**Review by zFernand0 — APPROVED** [review]

**Review by traeok — APPROVED** [review]

**Pull request merged**
